providers/amazon/src/airflow/providers/amazon/aws/sensors/bedrock.py

```diff
@@ -34,6 +36,9 @@
 from airflow.providers.amazon.aws.utils.mixins import aws_template_fields
 
 if TYPE_CHECKING:
+    from airflow.providers.amazon.src.airflow.providers.amazon.aws.triggers.bedrock import (
```

Comment:
comment to block merge - i just noticed the bad import path in the type checking block; fix coming.

We have a precommit/static check for paths in the top-level that would have caught that but it looks like maybe it ignores imports int he type checking block?